### PR TITLE
[Issue 683] Static site public launch deliverable spec

### LINKED ISSUE #683: [Deliverable spec]: Static site public launch
### Description

Create a deliverable spec for site static public launch.

### Definition of Done

- [ ] Milestone document is updated and stored in `documents/milestones/individual_milestones/`
- [ ] The milestone document has been linked under the appropriate section in the short description doc
- [ ] The milestone has node has been updated to "planning" in the dependency diagram
- [ ] Milestone has been reviewed and approved by the internal team
- [ ] A corresponding epic has been created for the milestone

### PR DESCRIPTION
**Pull request by widal001**
## Summary
Creates a deliverable spec for #692 

Fixes #683 

### Time to review: __15 mins__

Added the [`static_site_public_launch.md`](https://github.com/HHS/simpler-grants-gov/blob/issue-683-static-site-public-launch-deliverable-spec/documentation/milestones/individual_milestones/static_site_public_launch.md) deliverable spec.

It may be easier to view the [rendered version of this deliverable spec](https://github.com/HHS/simpler-grants-gov/blob/issue-683-static-site-public-launch-deliverable-spec/documentation/milestones/individual_milestones/static_site_public_launch.md)

### CODE REVIEW

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+- A communications plan that outlines future plans for stakeholder engagement about the Simpler.Grants.gov initiative. This plan should answer the following questions:
+  - Which stakeholder groups are we planning to engage?
+  - Who is responsible for managing the relationship with each group?
+  - What is the primary channel and cadence for those communications?
+
+Both the initial email communication and the communication plan should be reviewed and approved by key stakeholders in the recurring communications meetings.
+
+### Email subscription mechanism
+
+One stretch goal for this deliverable is to adopt a service that allows site visitors to subscribe to a mailing list to receive updates about the Simpler.Grants.gov initiative. As part of this deliverable the team should answer the following questions:
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md:115]: The email signup is part of the draft content we're circulating for review, so I don't think it's a stretch goal any more and is rather a core goal. 

Additional, the Figma prototypes include a survey tool for signing up to be a user research participant, so we should add that here as well.

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] The static site has been updated with content approved by internal stakeholders
+  - [ ] The front-end infrastructure can handle site traffic of up to 1,000 concurrent users
+  - [ ] Tickets and/or documentation have been created that describe the infrastructure changes needed to support future site traffic at the following levels:
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md:135]: Hmm, I'm wondering if this is essential or more of a stretch goal?

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+5. Site availability
+6. [Lighthouse score](lighthouse) for the site
+7. Deployment build time
+8. Deployment/hosting costs
+9. (Stretch) Number of visitors who subscribe to the mailing list
+
+### Destination for live updating metrics
+<!-- Required -->
+
+Page on the public wiki. **Note:** This will likely change once we deliver [the public measurement dashboard](https://github.com/HHS/simpler-grants-gov/issues/65)
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md:169]: we can also put a link to our page on https://analytics.usa.gov/

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+  - [ ] Engineering staff are notified when the site goes down
+  - [ ] Engineering staff have been trained on a documented incident response plan that describes a set of actions to take when the site goes down
+  - [ ] A set of SLAs have been defined internally for site availability and responsiveness
+- [ ] Communications requirements
+  - [ ] An email has been sent out to the subset of public stakeholders with whom we plan to share the static site
+  - [ ] Key internal stakeholders (e.g. help desk staff, HHS leadership) have been notified when the email is sent out so that they can prepare for questions from public stakeholders
+  - [ ] A longer-term stakeholder engagement plan has been drafted that describes future phases of engagement
+  - [ ] Help Desk is notified and trained for any potential support issues that may come through
+
+#### Nice to have
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md:148]: Same comment as above: should we add "must haves" for the list serv as well as for a survey tool for signing people up for user research?

**Review by lucasmbrown-usds — CHANGES_REQUESTED** [review]
Overall looking good! A few comments.

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md (hunk 2 above)
Comment (widal001) [reply documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md]: Agreed, it's not mission critical for the launch. I moved this down into the "nice-to-have" section in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md (hunk 3 above)
Comment (widal001) [reply documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md]: I like that too! I added it as another option in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)

File: documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md (hunk 4 above)
Comment (widal001) [reply documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md:146]: Sure, I added these two additional criteria in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/335745fcd28577b6f0d3b40ead9d58c243fda091) and [the previous commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+<!-- Required -->
+
+- **What:** We want to make a series of updates to the content and the infrastructure to the static site so that we are prepared to launch to a wider audience publicly and effectively communicate the project's goals, ways of working, and ongoing efforts.  
+- **Why:** Continues to improve front-end infrastructure, demonstrate ongoing value both internally and publicly, and begins to build public excitement for the Simpler Grants.gov initiative. These updates underscore our agile methodology of releasing often, even with minimal updates. We also want to ensure that our site can handle an increase in traffic. 
+- **Who:** The site will be announced directly with a small group of applicants, grantors, and internal stakeholders. These include: 
+    - Federal Demonstration Partnership (mostly applicants)
+    - S2S Federal User Group (grantors) 
+    - Grantor S2S CGMO (Grants Management Officers and others within HHS)
+    - Internal content managers
+    - Subset of grants.gov mailing list
```
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md:24]: Oh I thought the groups listed above _were_ a subset of the mailing list. I'll take this out!

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] The static site has been updated with content approved by internal stakeholders
+  - [ ] The front-end infrastructure can handle site traffic of up to 1,000 concurrent users
+  - [ ] Engineering staff are notified when the site goes down
+  - [ ] Engineering staff have been trained on a documented incident response plan that describes a set of actions to take when the site goes down
+- [ ] Communications requirements
+  - [ ] An email has been sent out to the subset of public stakeholders with whom we plan to share the static site
+  - [ ] Key internal stakeholders (e.g. help desk staff, HHS leadership) have been notified when the email is sent out so that they can prepare for questions from public stakeholders
+  - [ ] A longer-term stakeholder engagement plan has been drafted that describes future phases of engagement
```
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Sure thing! Updated this in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/a52d00e8725b9a589bdb55f884f2c4f71a018b45)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Updated this in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/a52d00e8725b9a589bdb55f884f2c4f71a018b45)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+  - to approve the content on the site before we share it with the public, so that I know what information will be visible to external stakeholders.
+  - to know when I can share the site publicly, so that I can direct key stakeholders to a centralized location where they can learn more about the Simpler.Grants.gov initiative.
+  - to have a communications strategy for stakeholder engagement, so that we have clear expectations about which groups will receive updates on the Simpler.Grants.gov initiative and can review proposed messaging to those groups.
+  - clear and concise content that effectively communicates our project's goals, vision, and way of working to the public. This user story is essential to ensure that we uphold our commitment to transparency, fostering open communication and alignment with our overarching project objectives.
+- As an **internal developer**, I want:
+  - to be notified when the site goes down, so that I can work to troubleshoot the issue and minimize downtime.
+  - the front-end infrastructure to auto-scale based on site traffic, so that I don't have to manually monitor traffic and provision additional resources when there is heavy usage.
```
[the remark this answers is not in the document]
Comment (lucasmbrown-usds) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md:51]: from sumi: 

>front-end infrastructure is able to handle the expected traffic with limited/no performance degradation

[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Good call, reworded this using @sumiat's suggested language in this commit: [fix: Incorporates PR feedback pt. 2](https://github.com/HHS/simpler-grants-gov/pull/693/commits/6b7a316b2a2cb273336d6701f9c2e96c40e198ac)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+  - Internally-defined service level agreements for availability and incident response
+  - Documented incident response plan with steps and roles identified
+  - Documented plan for receiving and responding to alerts
+  - Staff trainings, tabletop and/or dry-runs for common types of incidents
+
+### Communication strategy
+
+The two main communication outputs needed for this deliverable are:
+
+- An email campaign that is sent to the subset of the grants.gov mailing list with whom we plan to share the site.
```
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Good catch! I fixed this in this commit: [fix: Incorporates PR feedback pt. 2](https://github.com/HHS/simpler-grants-gov/pull/693/commits/6b7a316b2a2cb273336d6701f9c2e96c40e198ac)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+- A communications plan that outlines future plans for stakeholder engagement about the Simpler.Grants.gov initiative. This plan should answer the following questions:
```
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Updated the language in this commit: [fix: Incorporates PR feedback pt. 2](https://github.com/HHS/simpler-grants-gov/pull/693/commits/6b7a316b2a2cb273336d6701f9c2e96c40e198ac)

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 10]
```diff
@@ -0,0 +1,258 @@
...
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
```
[the remark this answers is not in the document]
Comment (widal001) [reply documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md]: Checked these off in this commit: [fix: Incorporates PR feedback pt. 2](https://github.com/HHS/simpler-grants-gov/pull/693/commits/6b7a316b2a2cb273336d6701f9c2e96c40e198ac)

We discussed doing the same for other deliverable specs that we've completed once they've been delivered.

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 11]
```diff
@@ -0,0 +1,256 @@
...
+
+- **Systems**
+  - Heartbeat monitoring of internal services and public urls to ensure that the website is live and publicly accessible
+  - Automated notifications when:
+    - services are down
+    - 4xx or 5xx requests reach a certain threshold
+    - resources reach a certain usage threshold or are auto-scaled
+  - Logging capabilities (e.g. search, filtering) and retention policies to assist with troubleshooting
+- **Procedures**
+  - Internally-defined service level agreements for availability and incident response
```
Comment (sumiat) [inline documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md:95]:
```suggestion
```

**Review by sarahknoppA6 — APPROVED** [review]
Looks very thorough and well thought out!

**Comment by EOKENAVA** [comment]
Great job Billy! Looks good

File: documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md [hunk 12]
```diff
@@ -92,17 +92,15 @@ In particular, it would be helpful to review our approach to the following:
...
 - **Procedures**
-  - Internally-defined service level agreements for availability and incident response
   - Documented incident response plan with steps and roles identified
   - Documented plan for receiving and responding to alerts
   - Staff trainings, tabletop and/or dry-runs for common types of incidents
 
 ### Communication strategy
 
-The two main communication outputs needed for this deliverable are:
+The two main communication outputs needed for this deliverable is an email campaign that is sent to the people with whom we plan to share the site.
```
Comment (lucasmbrown-usds) [inline documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md:101]: nit: `The two main communication outputs` should now be `The main communication output'

**Pull request merged**